core/src/initializers/settings.ts

```diff
@@ -171,6 +171,14 @@ export class Plugins extends Initializer {
       },
     ];
 
+    const settingKeys: string[] = [
+      ...coreSettings,
+      ...interfaceSettings,
+      ...telemetrySettings,
+    ].map(({ key }) => key);
```

Comment:
not a major thing but i think there is `.keys()` that will do this.

Reply:
In this case, I believe [`.keys()` would return `[0, 1, 2, ...]`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/keys).

This is an array of objects, and I'm pulling out the `key` _property_ from each one.

Reply:
Ah, I see. brackets not `{` and I misunderstood what was in `telemetrySettings`. good work!